src/ServicePulse.Host/vue/src/components/TimeSince.vue

```diff
@@ -14,10 +14,17 @@ const title = ref(),
   text = ref();
 
 function updateText() {
-  const m = moment.utc(props.dateUtc);
-  text.value = m.fromNow();
-  title.value =
-    m.local().format("LLLL") + " (local)\n" + m.utc().format("LLLL") + " (UTC)";
+  if (props.dateUtc !== "0001-01-01T00:00:00") {
```

Comment:
Looks like Jayanthi also changed this https://github.com/Particular/ServicePulse/pull/1374/files#diff-202dfa9a63b84ec107514c05a40a11586af4a0805a5a1ca30ad7185d828e554d

Reply:
I talked with @soujay and we are going to merge this version of the `TimeSince.vue`.  I just updated the text and title values to "n/a" instead, so that it would make more sense in both cases.